docker/wrapper.py

```diff
@@ -56,8 +56,25 @@ def call_pipeline(mount, args):
         else:
             log.info('Flag "--no-clean" was used, therefore {} was not deleted.'.format(work_dir))
 
+    #DEBUG !!!
+    print("DEBUG!!!!! output dir is "+mount+" and files are:")
+    print(os.listdir(mount))
+    fail_files = [fail_file for fail_file in os.listdir(mount) if fail_file.startswith("FAIL.") and fail_file.endswith(".tar.gz")]
+    print("fail files:", ' '.join(fail_files))
+    cmd = ["mv", mount+"/"+fail_file, mount+"/"+fail_file[len("FAIL."):]]
```

Comment:
- This line is creating `cmd` based off a temporary variable, this is broken and will not do what you want. 
- Missing PEP convention for spaces around operators.
- Don't combine file paths with strings and slashes, it's not OS agnostic. Use `os.path.join`.
- Replace the string slice with `fail_file.lstrip('FAIL.')`
- If you make the `glob` change as suggested above, `mount` will already be part of the path for the elements in `fail_files`.